backend/mora/service/detail_reading.py

```diff
@@ -643,6 +675,12 @@ def convert(start, end, funcid, effect):
         }
         func[mapping.UUID] = funcid
 
+        if function == 'it':
```

Comment:
What I meant was, why not have this as a getter along the lines of `get_itsystem` and `get_unitid`, instead of having a very specific piece of code here, in an otherwise pretty generic function.